packages/neuron-ui/src/utils/ipc.ts

```diff
@@ -31,10 +31,17 @@ const sendCapacity = (address: string, capacity: string) => {
     capacity,
   })
 }
+const getHistory = (page: number, pageSize: number) => {
+  ipcRenderer.send(Channel.GetTransactions, {
+    page,
+    pageSize,
+  })
+}
 
 export default {
   asw,
   getLiveCell,
   getCellsByTypeHash,
   sendCapacity,
+  getHistory,
```

Comment:
Might be better to call this `getTransactions` instead of `getHistory`, but keep the UI link and label as `History`.

Just `Send` is for (human) user and `Transfer` is our component/module, `History` is for user to read while `Transactions` is the in app concept.

Reply:
[Updated](https://github.com/nervosnetwork/neuron/pull/92/commits/c7259636b1cbd30d79f6249025bd974f01686160)